x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/transform/action/GetTransformAction.java

```diff
@@ -132,11 +144,18 @@ public XContentBuilder toXContent(XContentBuilder builder, Params params) throws
                     invalidTransforms.size()
```

Comment:
That's a good point and actually the objects don't fit, right? 

"Invalid transforms" are still full config objects, only some part got somehow invalid, e.g. because a part in the DSL changed.

The "broken" transforms are an array of strings?